common/client.go

```diff
@@ -94,12 +108,44 @@ func (rc *RestClient) NewUrl(dest string) error {
 	return rc.modifyUrl(dest, nil)
 }
 
+// ListHosts list host
```

Comment:
Same here: We should say something like:

    ListHost queries the Topology service in order to return a list of currently configured hosts in a 
    Romana cluster.

Not just any hosts, you know.